srcpkgs/xbps-triggers/files/texmf-dist

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+#
+# Updates the shared-mime-info db file with update-mime-database(1).
+#
+# Arguments:	$ACTION = [run/targets]
+#		$TARGET = [post-install/post-remove]
+#		$PKGNAME
+#		$VERSION
+#		$UPDATE = [yes/no]
+#
+ACTION="$1"
+TARGET="$2"
+PKGNAME="$3"
+VERSION="$4"
+UPDATE="$5"
+
+texhash=usr/bin/texhash
+fmtutil=usr/bin/fmtutil-sys
+
+case "$ACTION" in
+targets)
+	echo "post-install post-remove"
+	;;
+run)
+	if [ ! -x ${texhash} ] && [ ! -x ${fmtutil} ]; then
+		exit 0
+	fi
+
+	echo "Updating texmf-dist hashes..." 
+	${texhash} 2>&1 >/dev/null || true # silence strange errors
+	echo "Updating texmf-dist formats..."
+	${fmtutil} --all >/dev/null || true
```

Comment:
With luatex-minimal, I get: *[ERROR]: not building luatex due to missing engine: luatex.*

Is it expected?

Reply:
Yes this is expected, because luatex is not installed, but its still in `fmtutil.cnf`. Maybe I should silence those errors? Or add a message at the beginning saying something like "Missing engine errors are due to missing packages"?